coldfront/templates/email/new_project_join_request.txt

```diff
@@ -1,6 +1,8 @@
 Dear managers of {{ project_name }},
 
-User {{ user_string }} has requested to join your project, {{ project_name }} via MyBRC user portal. Please approve/deny this request.
+User {{ user_string }} has requested to join your project, {{ project_name }} via the MyBRC User Portal.
+
+Please approve/deny this request here.
```

Comment:
In the (unlikely?) scenario that the user's browser doesn't support HTML emails, it'll fall back to this `.txt`, which won't actually have the URL. We should include it:

```
...this request here: {{ review_url }}.
```

(I imagine there are places in the code where this isn't the case, but we'll resolve those in #309.)

Reply:
made the change